src/main.zig

```diff
@@ -141,6 +143,24 @@ const Command = struct {
             else => err,
         };
 
+        // Calculate how many bytes are allocated inside `arena`.
+        // TODO This does not account for the fact that any allocations will be rounded up to the nearest page by `std.heap.page_allocator`.
+        var allocation_count: usize = 0;
+        var allocated_bytes: usize = 0;
```

Comment:
```suggestion
        var allocation_size: usize = 0;
```

So that both variables share the same prefix, with only the qualifier different. They line up in the source then, making the code more blocky and quicker to read/parse, with the qualifiers jumping out. We typically use `_size` to mean bytes.